book/src/faq.md

```diff
@@ -453,8 +454,22 @@ The settings are as follows:
    ```
 
    You can refer to [Redundancy](./redundancy.md) for more information.
-   
-   It is also worth noting that the `--beacon-nodes` flag can also be used for redundancy of beacon nodes. For example, let's say you have a beacon node and a validator client running on the same host, and a second beacon node on another server as a backup. In this case, you can use `lighthouse vc --beacon-nodes http://localhost:5052, http://local_IP:5052` on the validator client.
+
+2. If the beacon node and validator clients are on different servers *and different networks*, the settings are as follows:   
+
+    i. Beacon node:
+
+   The setting is the same as in scenario 1.
+
+    ii. Validator client:
+    Use the flag `--beacon-nodes` to point to the beacon node. However, since the beacon node and the validator client are on different networks, the IP address to use is the public IP address of the beacon node, i.e., `lighthouse vc --beacon-nodes http://public_IP:5052`. You can get the public IP address of the beacon node by running the command ` dig +short myip.opendns.com @resolver1.opendns.com` on the server running the beacon node.
+
+    Additionally, port forwarding of port 5052 on the router connected to the beacon node is required for the vc to connect to the bn. To do port forwarding, refer to [how to open ports](./advanced_networking.md#how-to-open-ports).
+
+
+Finally, in both cases, if you have firewall setup, e.g., `ufw`, you will need to allow port 5052 (assuming that the default port is used) with `sudo ufw allow 5052`.
```

Comment:
```suggestion
Finally, in both cases, if you have firewall setup, e.g., `ufw`, you will need to allow port 5052 (assuming that the default port is used) with `sudo ufw allow 5052`. As mentioned previously, if the beacon node API port is exposed to the public Internet then we suggest only allowing the IP address of the validator client to access port 5052 (assuming the default port it allowed).
```
 
 @chong-he perhaps you'd like to add the `ufw` command for only allowing a single IP address? Unfortunately if they do `sudo ufw allow 5052` it will still allow any IP address. Perhaps we could just always instruct the user to allow a specific IP address?

Reply:
> @chong-he perhaps you'd like to add the `ufw` command for only allowing a single IP address? Unfortunately if they do `sudo ufw allow 5052` it will still allow any IP address. Perhaps we could just always instruct the user to allow a specific IP address?

Added the info, thanks for the review! I only added the SSH tunneling guide as an alternative to port forwarding on the router. 

Same as above, didn't commit the suggestion directly but the revision has included the suggested changes.